assets/src/blocks/Counter/CounterFrontend.js

```diff
@@ -1,41 +1,41 @@
-import { Component, Fragment } from '@wordpress/element';
-import { getStyleFromClassName } from '../getStyleFromClassName';
+import {Component, Fragment} from '@wordpress/element';
+import {getStyleFromClassName} from '../getStyleFromClassName';
 
 export class CounterFrontend extends Component {
   constructor(props) {
     super(props);
     this.state = {
       remaining: 0,
-      completed: 0
+      completed: 0,
     };
 
     this.calculateRemaining = this.calculateRemaining.bind(this);
     this.getCounterText = this.getCounterText.bind(this);
   }
 
   componentDidMount() {
-    const { completed_api } = this.props;
+    const {completed_api} = this.props;
     // Calculate completed and remaining values depending on props
-    let counter = this;
+    const counter = this;
     counter.calculateRemaining();
     // Add an eventListener to the window to enable instantly updating counters with supported APIs
     if (completed_api && completed_api.startsWith('https://')) {
-      window.addEventListener('updateCounter', function () { counter.calculateRemaining(); }, false);
+      window.addEventListener('updateCounter', counter.calculateRemaining(), false);
```

Comment:
I think you need to remove the parenthesis, `counter.calculateRemaining()` => `counter.calculateRemaining`